packages/keystone/lib/Keystone/format-error.js

```diff
@@ -67,64 +63,23 @@ const flattenNestedErrors = error =>
     []
   );
 
-const formatError = error => {
+export const formatError = error => {
```

Comment:
You can't use ESM export unless you make the whole package use Preconstruct. (this is probably why the example smoke tests are hanging)

Reply:
Ah cool, just noticed they were being weird. Will investigate 👍